src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -161,4 +168,18 @@
 {0} - The managed type name
 {1} - The exception message of the associated exception</comment>
   </data>
+  <data name="XA4301" xml:space="preserve">
+    <value>APK already contains the item {0}; ignoring.</value>
+    <comment>The abbreviation "APK" should not be translated.
+{0} - The file name.</comment>
+  </data>
+  <data name="XA4301_ABI" xml:space="preserve">
+    <value>Cannot determine ABI of native library {0}.</value>
+    <comment>The abbreviation "ABI" should not be translated.</comment>
+  </data>
+  <data name="XA4301_ABI_Ignoring" xml:space="preserve">
+    <value>Could not determine ABI of some native libraries. Ignoring those: {0}</value>
+    <comment>The abbreviation "ABI" should not be translated.
+{0} - Comma-separated list of the ignored library file names.</comment>
+  </data>
```

Comment:
Similar to the XA0107 messages, I debated on splitting these XA4301 into different error codes, and for similar reasons, I left them all under one code for now.

Although the wording of the first `XA4301` message doesn't directly mention it, that message is about native libraries too, like the other messages.